prisma/schema.sql

```diff
@@ -228,6 +228,7 @@ CREATE TABLE application_processing (
   review_request_completed_updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
   invoice_number INTEGER UNIQUE,
   documents_url TEXT,
```

Comment:
we can remove the documents_url from the db schema since we will be generating a new one on every read